tests/load/pipeline/test_scd2.py

```diff
@@ -69,20 +71,6 @@ def strip_timezone(ts: datetime) -> datetime:
         return table
     return sorted(table, key=lambda d: d[sort_column])
 
-    return sorted(
```

Comment:
This removed block is not related to the PR. It's dead code I forgot to remove in another recent PR.